cmd/helper/fleet_helper.go

```diff
@@ -0,0 +1,73 @@
+package helper
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/loft-sh/devpod/cmd/flags"
+	"github.com/loft-sh/devpod/pkg/agent"
+	"github.com/spf13/cobra"
+)
+
+// FleetServerCmd holds the fleet server cmd flags
+type FleetServerCmd struct {
+	*flags.GlobalFlags
+
+	WorkspaceID string
+}
+
+// NewFleetServerCmd creates a new fleet command
+func NewFleetServerCmd(flags *flags.GlobalFlags) *cobra.Command {
```

Comment:
I'm not sure if this should be a separate command tbh. Is there something speaking against moving it into the regular daemon?